ocaml/libs/http-lib/xmlrpc_client.ml

```diff
@@ -51,7 +51,7 @@ let connect ?session_id ?task_id ?subtask_of path =
 let xmlrpc ?frame ?version ?keep_alive ?task_id ?cookie ?length ?auth
     ?subtask_of ?query ?body ?(tracing = None) path =
   let traceparent =
-    let open Tracing in
+    let open Tracing_core in
```

Comment:
I suggest you keep it `Tracing` instead of `Tracing_core`, so that you do not need to change so many files and therefore can significantly simplify this PR.